removal_from_ansible.rst

```diff
@@ -165,3 +165,58 @@ Re-adding collection to Ansible
 -------------------------------
 
 There is no simplified process. Once the collection has been removed from Ansible Y.0.0, it needs to go through the full inclusion process to be re-added to the Ansible package. Exceptions are only possible if the Steering Committee votes on them. The Steering Committee can approve or deny a fast re-entry without going through the full review process.
+
+Collections not satisfying the Collection Requirements
+======================================================
+
+A collection can be removed from the package if it violates one or more of the `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ without making enough effort to mitigate these violations.
+
+This section is not applicable to cases of broken or unmaintained collections. Instead, see the corresponding paragraphs of this document.
+
+Identifying and removing a collection
+-------------------------------------
+
+Conditions for removal
+~~~~~~~~~~~~~~~~~~~~~~
+
+#. A collection violates one or more of the `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_.
+#. Collection maintainers have not fixed the violations and have not released a fixed version of the collection within the time period established by this document.
+
+Process
+~~~~~~~
+
+#. A community person who found a violation creates an issue in the collection's repository. The issue contains:
+
+  * References to corresponding `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ the collection violates.
+  * Actions collection maintainers need to do to make the collection satisfying the requirements.
+#. A default term for the collection to solve the issue is four weeks since the issue was created. It can vary depending on a requirement violated, SC opinions or other circumstances.
+#. If the violation is not fixed or there is a disagreement between the reporter and the maintainers, the reporter or another person creates a `community topic <https://github.com/ansible-community/community-topics/issues>`_.
+#. Two SC members check the reported circumstances and confirm in the topic that the violation is present.
```

Comment:
I'm wondering whether we still need this, or in which form precisely. The basic idea is to have a 'sanity check' whether this is actually something that needs to be fixed in the collection, or whether it is a) a wrong interpretation of the rule, or b) rather the rule should be changed than the violation, or c) something else. This step should be deciding between "let's discuss this further" and "let's start a vote (with a binary result)". What do you all think, should this be updated/modified (if yes: how?) to explain that better?

Reply:
cc @mariolenz @gotmax23 ^

Reply:
Yeah, I kind of agree with @felixfontein. This is how we conduct all votes; there's a few days for folks to leave comments and discuss the merits, and then we start the vote if there's no significant opposition. I don't have strong feelings about whether this step should be changed or removed, though, so I'll defer to others.

Reply:
Yeah, I kind of agree with @felixfontein. This is how we conduct all votes; there's a few days for folks to leave comments and discuss the merits, and then we start the vote if there's no significant opposition. I don't have strong feelings about whether this step should be changed or removed, though, so I'll defer to others.